dataverse-webapp/src/main/webapp/editFilesFragment.xhtml

```diff
@@ -396,7 +373,7 @@
                                 <p:message for="fileDescription"/>
                             </ui:fragment>
                         </div>
-                        <div jsf:rendered="#{createDataset or EditDatafilesPage.mode eq 'UPLOAD' }">
+                        <div jsf:rendered="#{fileTermsOfUse}">
```

Comment:
I think name/param should be more descriptive, like `isUploadMode`;

Reply:
I've coupled params names with ids of elements they are used in. They idea was for them to be easily tracked during separation of Files tab.
But you're right it'll be better to give them more meaningful names.
We should however give names that are meaningful from both `EditDatafilesPage` and `DatasetPage` thus in this case `DatasetPage` is not aware of `mode eq 'UPLOAD'` so the name should be more like: `isUploadable` or `isUploadPermited` or such.